File: my-journey/dsa/week-01/002-multiples.cpp
// Problem: Multiples
// Pattern: Basics / Conditionals
// Link: https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/J
// Difficulty: Easy
// Time:  O(1)
// Space: O(1)

#include <iostream>
using namespace std;

void solve() {
    int a, b;
    cin >> a >> b;
    // your logic here
    if (a % b == 0 || b % a == 0)
        cout << "Multiples\n";
    else
        cout << "No Multiples\n";
}

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    solve();
    return 0;
}

File: my-journey/dsa/cp/week-01/python/002-multiples.py
# Problem: Multiples
# Pattern: Basics / Conditionals
# Link: https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/J
# Difficulty: Easy
# Time:  O(1)
# Space: O(1)

import sys
input = sys.stdin.readline


def solve():
    a, b = map(int, input().split())
    # your logic here
    if a % b == 0 or b % a == 0:
        print("Multiples")
    else:
        print("No Multiples")


if __name__ == "__main__":
    solve()
